### ISSUE TITLE
[Usable - Bot Webchat - Suggested actions ]: Next and previous buttons are getting overlapped with cards button.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4032]
User Experience:
If contents present on the page will not be visible properly and buttons will get overlapped then users with visual disabilities who depend on this feature will not be able to see the content and will not be able to perform action easily.

Repro Steps:
1. Open URL: Web Chat: Full-featured bundle (microsoft.github.io)
2. Web chat bundle will open.
3. Navigate on the web chat bot and enter "suggested-actions’" keyword in Type your message box, then select send button.
4. Cards related to the keyword will appear, navigate on the cards.
5. Check and verify that step 3-4 are accessible and MAS compliant.

Actual Result:
Next and previous buttons are getting overlapped with cards button.

Expected Result:
Next and previous buttons should not get overlapped with cards button.

Test Environment:
OS: Win 10 21H1
Build: OS Build 19043.1147
Browser: Edge Chromium Version 94.0.972.2 (Official build) dev (64-bit)
#URL: Web Chat: Full-featured bundle (microsoft.github.io)

**Comment by KattaBhargav** [comment]
#A11yUsable;#A11ySev2;#HCL;#Accessibility;#Benchmark;#Win10-Edge(Chromium);#E2E-HCL-BotFrameworkWebChat-Aug21;

**Comment by compulim** [comment]
Today, we have allocated spaces for the next/prev flipper buttons for the suggested actions carousel.

![image](https://user-images.githubusercontent.com/1622400/131763712-a5bac8f7-a87e-4480-ab25-ed85393dd287.png)

![image](https://user-images.githubusercontent.com/1622400/131763765-07078ae1-d9d9-425a-b1cc-a24b6d7abb81.png)

Need to clarify with the accessibility team if they are talking about the "temporal" or "brief" overlap as shown on the right side of the first screenshot, or the left side of the second screenshot.

**Comment by v-shikch** [comment]
We have added this issue in our SME discussion queue. Will take action after getting feedback from SME.

**Comment by v-shikch** [comment]
We have added this issue in our SME discussion queue. Will take action after getting feedback from SME.

**Comment by compulim** [comment]
Thanks @v-shikch. Look forward to you and SME feedbacks. Feel free to ping me on Teams for faster response.

**Comment by stevkan** [comment]
Closing due to lack of response. If you have a response/feedback to provide, please feel free to reopen the issue.
